features/org.wso2.msf4j.feature/pom.xml

```diff
@@ -64,9 +64,23 @@
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Snakeyaml dependency is from carbon config component. Have to bump the dependency there